### ISSUE TITLE
trouble building git-version-macro 0.3.0

### ISSUE THREAD

**Issue by droundy** [m-ou-se/rust-git-version#8]
Here is my error message:
```
Compiling git-version-macro v0.3.0                                           
error[E0658]: imports can only refer to extern crate names passed with `--extern` on stable channel (see issue #53130)
  --> /home/droundy/.cargo/registry/src/github.com-1ecc6299db9ec823/git-version-macro-0.3.0/src/lib.rs:11:5
   |                                                                            
10 | mod utils;                                                                 
   | ---------- not an extern crate passed with `--extern`                      
11 | use utils::{describe_cwd, git_dir_cwd, VERSION_ARGS};                      
   |     ^^^^^                                                                  
   |                                                                            
note: this import refers to the module defined here                             
  --> /home/droundy/.cargo/registry/src/github.com-1ecc6299db9ec823/git-version-macro-0.3.0/src/lib.rs:10:1
   |                                                                            
10 | mod utils;                                                                 
   | ^^^^^^^^^^                                                                 
                                                                                
error: aborting due to previous error                                           
                                                                                
For more information about this error, try `rustc --explain E0658`.             
error: Could not compile `git-version-macro`.                            
```
This is with
```
$ rustc --version
rustc 1.31.0 (abe02cefd 2018-12-04)
```
It built just fine on my other computer.  Is there a dependency on a very recent version of rust?

**Comment by m-ou-se** [comment]
I'm trying it now with 1.31.0...

**Comment by m-ou-se** [comment]
Yup, fails exactly the same.

Looks like we're using a feature added in 1.32: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1320-2019-01-17

> Module paths with no leading keyword like super, self, or crate, will now always resolve to the item (enum, struct, etc.) available in the module if present, before resolving to a external crate or an item the prelude.

**Comment by m-ou-se** [comment]
Published as 0.3.1!

**Comment by droundy** [comment]
Thanks so much for the incredibly quick fix!
